### PR TITLE
feat: organization enable and archive

### PR DESCRIPTION
**Pull request by Macroz**
Add organization enable and archive function for #2039 

- all items depend on organizations, even unused
- organizations don't depend on anything

# Definition of Done / Review checklist

## Reviewability
- [x] link to issue

## API
- [x] API is documented and shows up in Swagger UI
- [x] API is backwards compatible or completely new

## Documentation
- [ ] update changelog if necessary

## Testing
- [x] complex logic is unit tested
- [x] valuable features are integration / browser / acceptance tested automatically

### CODE REVIEW

File: src/clj/rems/api/schema.clj [hunk 1]
```diff
@@ -44,7 +44,9 @@
           (s/optional-key :organization/last-modified) DateTime
           (s/optional-key :organization/owners) [User]
           (s/optional-key :organization/review-emails) [{:name LocalizedString
-                                                         :email s/Str}]}))
+                                                         :email s/Str}]
+          (s/optional-key :organization/enabled) s/Bool
+          (s/optional-key :organization/archived) s/Bool}))
```
Comment (opqdonut) [inline src/clj/rems/api/schema.clj:49]: am I missing something or do existing organizations default to disabled and unarchived? wouldn't this need a migration?

Comment (Macroz) [reply src/clj/rems/api/schema.clj:49]: I don't think anyone is running new organization code yet so it should be ok? Also the disabling and archiving is not functional anywhere yet?

Comment (opqdonut) [reply src/clj/rems/api/schema.clj:49]: Ok fair enough. But there should probably still be defaults for enabled & archived when creating an organization, right?

Comment (Macroz) [reply src/clj/rems/api/schema.clj:49]: Do you mean this part?

https://github.com/CSCfi/rems/blob/master/src/clj/rems/db/organizations.clj#L16

Comment (opqdonut) [reply src/clj/rems/api/schema.clj:49]: Yeah!

**Pull request merged**